geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/KafkaDataStoreSchemaManager.scala

```diff
@@ -137,6 +138,38 @@ trait KafkaDataStoreSchemaManager extends DataStore with LazyLogging {
     }
   }
 
+  def updateKafkaSchema(typeName: String, sft: SimpleFeatureType) = {
```

Comment:
IIRC the issue we ran into with unit tests for this functionality is that we couldn't figure out how to get the keyword command to connect to the same mock data store as the unit test. Using the parameters, the keyword command would just spin up a new mock datastore without the keywords it needed to modify.